### PR TITLE
feat: add dataset collection endpoint, script to scrape validator db

### PR DESCRIPTION
**Pull request by jarvis8x7b**
context:
- each validator's DB may be super big, so jsonl dataset is chunked into 50mb parts before calling the validator API
- added `entrypoints/` folder for dataset service, unsure if we will use it for other things in the future. left one-off `scripts/extract_dataset.py` because it's a once in a while thing

### CODE REVIEW

File: entrypoints.sh [hunk 1]
```diff
@@ -73,4 +73,27 @@ if [ "$1" = 'validator' ]; then
...
     ${EXTRA_ARGS}
-fi
\ No newline at end of file
+fi
+
+if [ "$1" = 'extract-dataset' ]; then
+    echo "Environment variables:"
+    echo "WALLET_HOTKEY: ${WALLET_HOTKEY}"
+    echo "DATABASE_URL: ${DATABASE_URL}"
+    echo "DATASET_SERVICE_BASE_URL: ${DATASET_SERVICE_BASE_URL}"
```
Comment (codebender37) [inline entrypoints.sh:82]: what would be the `DATASET_SERVICE_BASE_URL` ?
Do we need to put that on `env.validator.example` ?

Comment (jarvis8x7b) [reply entrypoints.sh:82]: it's a separate service that we'll host so validators can just call. yep lemme add it

Comment (jarvis8x7b) [reply entrypoints.sh:82]: added

**Review by karootplx — APPROVED** [review]
lgtm

**Pull request merged**